src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -19,7 +19,7 @@ azure-keyvault==1.1.0
 azure-loganalytics==0.1.0
 azure-mgmt-advisor==9.0.0
 azure-mgmt-apimanagement==3.0.0
-azure-mgmt-appconfiguration==2.0.0
+azure-mgmt-appconfiguration==2.1.0b2
```

Comment:
Is there a stable version that has the changes we need? We normally dont take a dependency on pre-release versions.

Reply:
I'm syncing up with @shenmuxiaosen to see if we have any stable version with our required changes.

Reply:
We can use the preview version. As per my understanding with Shuai, the SDK team started marking preview version recently (i.e. `2.1.0bxx`). earlier they just didn't update the versions for minor versions.

Reply:
We used to release CLI based on some preview rp version. So I think it should be fine here.

Reply:
Okay, thanks for confirming!